README.md

```diff
@@ -1,34 +1,124 @@
 1. Assigning values to the variables.
-2. Choosing particular subject from several Aspect nodes for several decomposition of the system on same hierarchical level.
+2. Choosing particular subject from several Aspect nodes for several
+   decomposition of the system on same hierarchical level.
 3. Selecting one entity from various options of Specialization node.
 4. Specifying cardinality in Multi-Aspect node.
 
-# SESTools
+## SESEditor & PESEditor
+
+SESEditor for building System Entity Structures and PESEditor for interactive
+pruning. SESEditor which is designed in such a way that a user can draw SES
+graphs on the screen almost as one would on paper. An SES is represented by a
+directed tree structure. Here, objects are represented by nodes which are
+connected using edges. Elements icons are added in the toolbox for easy access.
+The vertices or elements and edges can be drawn by clicking and dragging the
+mouse. Also nodes and edges can be easily moved to any position. The drawing
+panel is synchronized with the bottom-left tree. If there is an element
+addition in one place, either in the white drawing panel or the left tree, it
+will be added in both the sections automatically. Variables can be attached to
+the nodes. Eventually, the attached variables are listed in the variable table
+on the right top corner during any node selection from either of the
+trees.Furthermore, selection constraints can be added to the aspect node to
+restrict the choices of entities. These constraints are specified using XPath.
+Like variables, constraints are also listed on the constraint table on the
+right side of the editor during aspect node selection.
 
-SESEditor for building System Entity Structures and PESEditor for interactive pruning. SESEditor which is designed in such a way that a user can draw SES graphs on the screen almost as one would on paper. An SES is represented by a directed tree structure. Here, objects are represented by nodes which are connected using edges. Elements icons are added in the toolbox for easy access. The vertices or elements and edges can be drawn by clicking and dragging the mouse. Also nodes and edges can be easily moved to any position. The drawing panel is synchronized with the bottom-left tree. If there is an element addition in one place, either in the white drawing panel or the left tree, it will be added in both the sections automatically. Variables can be attached to the nodes. Eventually, the attached variables are listed in the variable table on the right top corner during any node selection from either of the trees.Furthermore, selection constraints can be added to the aspect node to restrict the choices of entities. These constraints are specified using XPath. Like variables, constraints are also listed on the constraint table on the right side of the editor during aspect node selection.
+PES Editor has been developed as an interactive pruning tool. The user
```

Comment:
Why do you change the format? Is that not autowrap?

Reply:
Because the linter tells us to. And I think it makes sense, otherwise the diff algorithm will not work well. The problem is, that the fundamental barrier between changes is lines. If one paragraph is only one line, two changes in the same paragraph will always create a nasty merge conflict.